nginx/public/node/frontend/public/js/app/search/chant-search/about-volpiano.template.html

```diff
@@ -1 +1,90 @@
-More information about volpiano goes here
\ No newline at end of file
+<div class="container-fluid">
+    <div class="row">
+        <p>
+            Volpiano is a font that was created for the transcription of chant
+            melodies. It was developed Fabian Weber at the University of
+            Regensburg under the direction of David Hiley.
+        </p>
+        <p>
+            With Volpiano, alphanumeric characters and other symbols are
+            rendered as pitches and other musical elements on standard staffs.
+            On Cantus Ultimus, you can use Volpiano to enter a certain melody
+            and search for that melody in available manuscripts.
+        </p>
+
+
+        <p><b>Volpiano Dictionary</b></p>
+
+        <p>Queries can be entered using the characters below.</p>
+    </div>
+    <div class="row justify-content-center" style="margin-bottom:20pt">
+        <div class="volpiano" style="text-align:center">
+            <div class="volpiano-syllable">1---</div><div
+                class="volpiano-syllable">e--<span
+                    class="volpiano-text-lg">e</span></div><div
+                class="volpiano-syllable">f--<span
+                    class="volpiano-text-lg">f</span></div><div
+                class="volpiano-syllable">g--<span
+                    class="volpiano-text-lg">g</span></div><div
+                class="volpiano-syllable">h--<span
+                    class="volpiano-text-lg">h</span></div><div
+                class="volpiano-syllable">j--<span
+                    class="volpiano-text-lg">j</span></div><div
+                class="volpiano-syllable">k--<span
+                    class="volpiano-text-lg">k</span></div><div
+                class="volpiano-syllable">l--<span
+                    class="volpiano-text-lg">l</span></div><div
+                class="volpiano-syllable">i--<span
+                    class="volpiano-text-lg">i</span></div><div
+                class="volpiano-syllable">I--<span
+                    class="volpiano-text-lg">I</span></div>
+        </div>
+    </div>
+    <div class="row">
+        <p><b>Features of Volpiano Search</b></p>
+        <ol>
+            <li>
+                There are two types of Volpiano search in Cantus Ultimus:
+                "Volpiano" and "Volpiano (Literal)." "Volpiano (Literal)"
+                matches the way pitches in the query are divided into neumes. In
+                Volpiano, a hyphen ("-") separates pitches belonging to
+                different neumes. For example, the two queries below will yield
```

Comment:
"the two queries below": I'm only seeing one query, `1---fgfh---33---fg-fh--` ...?

Reply:
Good point, let me add them